src/frontend/gatsby/src/components/SearchPage/SearchResults.jsx

```diff
@@ -41,14 +64,15 @@ const SearchResults = ({ text, filter }) => {
     return (
       <Container className={classes.searchResults}>
         <Box className={classes.root} boxShadow={2} marginTop={10}>
-          <ListSubheader>
-            <Typography variant="h1" color="secondary" className={classes.title}>
-              <Grid container className={classes.error}>
-                {' '}
+          <div className={classes.errorBackground}>
+            <h1>
+              500{' '}
+              <p className={classes.messageBox}>
```

Comment:
a `<p>` isn't allowed as a child of `<h1>`.  I would get rid of the `<h1>` and to everything with styling that you need.